core-bundle/src/Search/Indexer/DefaultIndexer.php

```diff
@@ -72,7 +72,6 @@ public function index(Document $document): void
             'protected' => false,
             'groups' => [],
             'pageId' => 0,
-            'noSearch' => true, // Causes the indexer to skip this document if there is no json-ld data
```

Comment:
I think the default value should be `false` here (default of a page object).

Reply:
We do not need a default value. There is an `isset()` check below.